odl/contrib/fom/test/test_supervised.py

```diff
@@ -68,5 +68,62 @@ def test_psnr(space):
     assert result == pytest.approx(expected)
 
 
+def test_mean_value_difference_sign():
+    space = odl.uniform_discr(0, 1, 10)
+    I0 = space.one()
+    I1 = -I0.copy()
+    assert np.abs(odl.contrib.fom.mean_value_difference(I0, I1)) > 0.1
```

Comment:
we should do `from odl.contrib import fom` up top to make these lines much more readable.

Reply:
Further, in this case we know the exact answer, right? It should be 2.0, so test against that.

```
assert fom.mean_value_difference(I0, -I0) == pytest.approx(2.0)
```